Resources/views/form/image_widget.html.twig

```diff
@@ -2,8 +2,8 @@
 
 {% block image_widget %}
 {% spaceless %}
-    <div class="cropper" data-cropper-options="{{ form.vars.cropper_options }}">
-        <div class="cropper-canvas-container{% if form.delete is defined %} cropper-canvas-has-delete{% endif %}" data-max-width="{{ max_width }}" data-max-height="{{ max_height }}">
+    <div class="cropper" data-cropper-options="{{ form.vars.cropper_options }}" data-max-width="{{ max_width }}" data-max-height="{{ max_height }}">
+        <div class="cropper-canvas-container{% if form.delete is defined %} cropper-canvas-has-delete{% endif %}" data-preview-width="{{ preview_width }}" data-preview-height="{{ preview_height }}">
             {% if form.vars.download_uri %}
                 <img src="{{ asset(form.vars.download_uri) }}">
```

Comment:
Could you please use the `{{ preview_width }}` and `{{ preview_height }}` to set max `<img>` width and height so that the preview is properly displayed when editing an existing image?

You can do something like that: 
```
<img src="{{ asset(form.vars.download_uri) }}" style="max-width: {{ preview_width }}px; max-height: {{ preview_height }}px;">
```